about-the-studio.html

```diff
@@ -33,31 +33,29 @@
           <div class="logo-bar">
             <div class="inner-wrapper">
               <div class="logo-bar-image">
-                <a href="#">
+                <a href="index.html">
                   <img src="assets/images/logo_white.png">
                 </a>
               </div>
               <nav class="logo-bar-nav small-heading">
-                <a href="#">Projects</a>
-                <a href="#">About the Studio</a>
+                <a href="project-subpage.html">Projects</a>
```

Comment:
to subpage jest niepodtrzebne
poza tym to jest podstrona listy projektów, więc liczba mnoga

zmień na projects.html